ansible_collections/arista/avd/roles/eos_designs/doc/network-services.md

```diff
@@ -361,6 +361,13 @@ mac_address_table:
                   hash_algorithm: < md5 | sha1 | sha256 | sha384 | sha512, Default -> sha512 >
                   key: < key password >
 
+            # Structured configuration and eos cli commands rendered on router_bgp.vlans
+            # This configuration will not be applied to vlan aware bundles
+            bgp:
```

Comment:
This feature should also be implemented for l2vlans. Can we do it in the same PR?